docs/design/features/runtime-handled-tasks.md

```diff
@@ -392,7 +392,7 @@ static Task<int> FooAdapter(int a, int b)
     try
     {
         result = Foo(a, b);
-        continuation = StubHelpers.Async2CallContinuation();
+        continuation = StubHelpers.AsyncCallContinuation();
     }
     catch (Exception ex)
     {
```

Comment:
This document has a number of places where we use "async2" referring to runtime async as opposed to "async1" referring to Task-based Roslyn generated async. I have left those occurrences in.

Reply:
Those places are mainly (or only) when describing adapters between runtime async and `Task<T>`, so perhaps some other terminology would still be better here.